pandas/tests/frame/test_repr_info.py

```diff
@@ -381,3 +381,10 @@ def test_info_memory_usage(self):
         # deep=True, and add on some GC overhead
         diff = df.memory_usage(deep=True).sum() - sys.getsizeof(df)
         self.assertTrue(abs(diff) < 100)
+
+    def test_info_empty_frame(self):
+        io = StringIO()
+        pd.DataFrame().info(buf=io)
+        assert (io.getvalue() == "<class 'pandas.core.frame.DataFrame'>\n"
+                                 "Index: 0 entries\n"
+                                 "Empty DataFrame")
```

Comment:
This test already works for me with latest master (and latest release). So I don't think this is testing what you changed in the code